crates/wasm-compose/example/README.md

```diff
@@ -99,14 +99,14 @@ To build the `service` component, use `cargo component build`:
 
 ```sh
 cd service
-cargo component build --release
+cargo component build --release --target wasm32-wasi
```

Comment:
The default target should already be `wasm32-wasi`.

```suggestion
cargo component build --release
```

Reply:
Doh! I completely overlooked that. Thanks